code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -125,6 +168,41 @@
 	..()
 	return TRUE
 
+/datum/reagent/medicine/C2/hercuri
+	name = "Hercuri"
+	description = "Not to be confused with element Mercury, this medicine excels in reverting effects of dangerous high-temperature environments. Prolonged exposure can cause hypothermia."
+	reagent_state = LIQUID
+	color = "#F7FFA5"
+	overdose_threshold = 25
+	reagent_weight = 0.6
+
+/datum/reagent/medicine/C2/hercuri/on_mob_life(mob/living/carbon/M)
+	if(M.getFireLoss() > 50)
+		M.adjustFireLoss(-2*REM, FALSE)
+	else
+		M.adjustFireLoss(-1.25*REM, FALSE)
+	M.adjust_bodytemperature(rand(-25,-5)*(TEMPERATURE_DAMAGE_COEFFICIENT*REM), 50)
+	M.reagents?.chem_temp +=(-10*REM)
```

Comment:
can this be used to bring chems below 0K?

Reply:
chem temp is effected by bodytemp so I haven't been able to get it below in practical testing, but It's theoretically possible.